breathecode/authenticate/tests/urls/tests_github.py

```diff
@@ -18,11 +18,9 @@ def test_github_without_url(self):
 
         data = response.data
         details = data['details']
-        status_code = data['status_code']
 
         self.assertEqual(2, len(data))
         self.assertEqual(details, 'No callback URL specified')
-        self.assertEqual(status_code, status.HTTP_500_INTERNAL_SERVER_ERROR)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
Should be better migrate this error to `ValidationException` because in the body its say is a error 500 and in the status code says is 400